tools/nnicmd/nnictl.py

```diff
@@ -39,87 +39,86 @@ def parse_args():
     # parse start command
     parser_start = subparsers.add_parser('create', help='create a new experiment')
     parser_start.add_argument('--config', '-c', required=True, dest='config', help='the path of yaml config file')
-    parser_start.add_argument('--webuiport', '-w', default=8080, dest='webuiport')
+    parser_start.add_argument('--port', '-p', default=51188, dest='port', help='the port of restful server')
     parser_start.set_defaults(func=create_experiment)
 
     # parse resume command
     parser_resume = subparsers.add_parser('resume', help='resume a new experiment')
     parser_resume.add_argument('--experiment', '-e', dest='id', help='ID of the experiment you want to resume')
     parser_resume.add_argument('--manager', '-m', default='nnimanager', dest='manager')
-    parser_resume.add_argument('--webuiport', '-w', default=8080, dest='webuiport')
+    parser_resume.add_argument('--port', '-p', default=51188, dest='port', help='the port of restful server')
     parser_resume.set_defaults(func=resume_experiment)
 
     # parse update command
     parser_updater = subparsers.add_parser('update', help='update the experiment')
     #add subparsers for parser_updater
     parser_updater_subparsers = parser_updater.add_subparsers()
     parser_updater_searchspace = parser_updater_subparsers.add_parser('searchspace', help='update searchspace')
+    parser_updater_searchspace.add_argument('--port', '-p', default=51188, dest='port', help='the port of restful server')
```

Comment:
how about creating a constant variable for 51188?

Reply:
ok, fixed in https://github.com/Microsoft/nni/pull/202